source/topics/troubleshooting/mysql-connection-refused.md

```diff
@@ -0,0 +1,29 @@
+If you run `aptible db:tunnel $DB_HANDLE` and try connecting to the db with the `mysql` commandline client, you may see the folowing error:
```

Comment:
This applies to both `aptible db:tunnel` but also general connections from an app to an Aptible MySQL database, so I'd mention both cases.